bitcoin/Cargo.toml

```diff
@@ -48,11 +48,12 @@ actual-serde = { package = "serde", version = "1.0.103", default-features = fals
 hashbrown = { version = "0.8", optional = true }
 
 [dev-dependencies]
-serde_json = "1.0.0"
+serde_json = "1.0.91"
 serde_test = "1.0.19"
 serde_derive = "1.0.103"
 secp256k1 = { version = "0.24.0", features = [ "recovery", "rand-std" ] }
 bincode = "1.3.1"
+mutagen = { path = "/home/tobin/build/github.com/llogiq/mutagen/mutagen" }
```

Comment:
I guess you wanted to use git dep :)

Reply:
It didn't work at first with git dependency, I just got it working as quickly as possible - thanks though, will fix before doing proper PR.